src/assets/src/components/common.tsx

```diff
@@ -86,10 +86,10 @@ export const LoadingDisplay: React.FC<LoadingDisplayProps> = (props) => {
 }
 
 
-export const showConfirmation = (dialog: React.RefObject<Dialog>, action: () => void, title: string, actionDescription: string) => {
+export const showConfirmation = (dialog: React.RefObject<Dialog>, action: () => void, title: string, description: string) => {
     dialog.current!.show({
         title: title,
-        body: `Are you sure you want to ${actionDescription}?`,
+        body: description,
```

Comment:
I elected to remove the "Are you sure you want to " prefix here and add it as part of the strings accompanying `showConfirmation` usages. I realize that introduces more repetition of language throughout, and I could see this being the wrong call. It just felt a little limiting for this function to require you to use that language every time, and also made it a little harder to understand what was happening when looking at any individual usage of "showConfirmation".

Reply:
I guess I missed initially that it also included the trailing `?`. We definitely would need to modify that so that we could include multiple sentences.